### PR TITLE
[feat] Added restart policy to docker compose & edit readme

### PR DESCRIPTION
**Pull request by sarunnut1236**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- PostgresQL
+- Grafana
+- Nginx Load Balancer
+
+## Getting Started
+
+To get started with the IntaniaOverflow microservices, follow these steps:
+
+1. **Set up the environment:**
+    Ensure you have Docker installed and configured on your machine. And find your own ```.env``` files for each services.
```
Comment (ThanatWongsamut) [inline readme.md]: Are we already combined it, may be can change in next PR

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Services
+- API Gateway
+- Asset Service
+- Logging Service
+- Notification Service
+- Reply Service
+- Thread Service
+- User Service
+- Voting Service
+- MongoDB
```
Comment (ThanatWongsamut) [inline readme.md]: Don't have MongoDB la

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+- PostgresQL
+- Grafana
+- Nginx Load Balancer
```
Comment (ThanatWongsamut) [inline readme.md:78]: Missing rabbitMQ?

**Review by ThanatWongsamut — COMMENTED** [review]
Left some comment

File: readme.md (hunk 1 above)
Comment (sarunnut1236) [reply readme.md]: pls check again

File: readme.md (hunk 2 above)
Comment (sarunnut1236) [reply readme.md]: deleted

File: readme.md (hunk 3 above)
Comment (sarunnut1236) [reply readme.md:78]: added

**Review by ThanatWongsamut — APPROVED** [review]

**Pull request merged**